beets/importer.py

```diff
@@ -672,7 +673,7 @@ def manipulate_files(self, move=False, copy=False, write=False,
                     # old paths.
                     item.move(copy, link)
 
-            if write and self.apply:
+            if write and (self.apply or self.choice_flag == action.RETAG):
```

Comment:
Yep, that's exactly what I meant. I also don't have a strong opinion; I was just noticing that we were checking `flag in (APPLY, RETAG)` quite often, so it might make sense to use a shortcut.